src/briefcase/platforms/windows/msi.py

```diff
@@ -43,7 +43,7 @@ def support_package_url_query(self):
         return [
             ('platform', self.platform),
             ('version', self.python_version_tag),
-            ('arch', "amd64" if (struct.calcsize("P") * 8) == 64 else "win32"),
+            ('arch', "amd64" if struct.calcsize("P") == 8 else "win32"),
```

Comment:
This one is slightly deliberate - yes, the end result is "==8", but the real check is "is this a 64 bit pointer".